client/blocks/importer/wordpress/utils.js

```diff
@@ -1,6 +1,15 @@
 export const SESSION_STORAGE_IS_MIGRATE_FROM_WP = 'is_migrate_from_wp';
 export const SESSION_STORAGE_MIGRATION_STATUS = 'migration_status';
 
+function ignoreFatalsForSessionStorage( callback ) {
+	try {
+		return callback();
+	} catch {
+		// Do nothing.
+		return undefined;
+	}
+}
```

Comment:
Could you expand a little on this @agrullon95?

Reply:
@escapemanuele - I added the comment from the original function I copied here. Its used to ignore exception logged in sentry for session storage usage incase the browser has issues. More context:  https://github.com/Automattic/wp-calypso/pull/79713

Reply:
Thank you, I was not aware of this problem.